pkg/cri/resource-manager/policy/builtin/memtier/pools.go

```diff
@@ -837,6 +838,24 @@ func (p *policy) compareScores(request Request, pools []Node, scores map[int]Sco
 	return id1 < id2
 }
 
+// affinityScore calculate the 'goodness' of the affinity for a node.
```

Comment:
Hey, one more comment: you have a very good description of what this is about in the commit message. Could/should that somehow be copy-pasted or referenced here, too(?)

Reply:
I'll do that.